templates/vue-demo-store/pages/checkout/index.vue

```diff
@@ -74,6 +74,7 @@ const selectedShippingMethod = computed({
   async set(shippingMethodId: string) {
     isLoading[shippingMethodId] = true;
     await setShippingMethod({ id: shippingMethodId });
+    await refreshShoppingAndPaymentMethods({ skipShipping: true });
```

Comment:
skipping makes the logic much more complicated, wouldn't be simpler to have two separate methods and just calling what you need?